scripts/upgrade

```diff
@@ -31,6 +31,11 @@ upstream_version=$(ynh_app_upstream_version)
 #=================================================
 
 domain=$(ynh_app_setting_get $app special_domain)
+server_name=$(ynh_app_setting_get $app server_name)
+if [ -z $server_name ]; then     # for retro compatibility
+   server_name=$domain
+   ynh_app_setting_set $app server_name  $domain
+fi
```

Comment:
Sorry, can you put this in line 63

Reply:
I tried upgrading from the actual master branch and it worked well